community.md

```diff
@@ -28,3 +28,9 @@ to the repository of the relevant component at the [bblfsh organization](https:/
 
 If you want to request a new feature, feel free to open it as an issue at the
 relevant GitHub repository.
+
+For deeper changes we use the [Babelfish Improvement Proposals](proposals/README.md).
+If the feature you want to request is quite complex
```

Comment:
I preferred to stay simple (that's something the BIP1 already explains), but maybe not the best wording.